client/src/pages/AddReview/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import propTypes from 'prop-types';
+// https://www.npmjs.com/package/react-typeform-embed
```

Comment:
people know where to look it up so kinda unnecessary ;)

Reply:
idk i dont mind seeing such links in code, we are beginners and a small team, and not really 'employed'
its comments/preference

Reply:
yeah no big deal, it's just I don't see a reason for a link to npm, where all of the packages come from..